app/controllers/vouchers_controller.rb

```diff
@@ -83,18 +83,22 @@ def create
     if shouldEmail
       email_confirmation(:confirm_reservation, @customer, theshowdate, thenumtoadd)
     end
-    
+
     redirect_to customer_path(@customer)
   end
 
   def update_comment
-    @voucher.update_attributes(:comments => params[:comments], :processed_by => current_user)
+    vchs = Voucher.find(params[:voucher_ids].split(","))
```

Comment:
do you mind make this more readable,like voucher.

Reply:
vchs is pretty readable, just an abbreviated version of vouchers. Fox uses it in his code as well.